infra/build_specified_commit.py

```diff
@@ -30,6 +30,37 @@
 BuildData = collections.namedtuple(
     'BuildData', ['project_name', 'engine', 'sanitizer', 'architecture'])
 
+_GIT_DIR_MARKER = 'gitdir: '
+
+
+def _make_gitdirs_relative(src_dir):
+  """Make gitdirs relative."""
+  for root_dir, _, files in os.walk(src_dir):
+    for filename in files:
+      if filename != '.git':
+        continue
+
+      file_path = os.path.join(root_dir, filename)
+      with open(file_path) as handle:
+        lines = handle.readlines()
+
+      new_lines = []
+      for line in lines:
+        if line.startswith(_GIT_DIR_MARKER):
+          absolute_path = line[len(_GIT_DIR_MARKER):].strip()
+          current_dir = os.path.dirname(file_path)
+          relative_path = os.path.relpath(absolute_path,
+                                          current_dir.replace(src_dir, '/src'))
```

Comment:
move current_dir.replace(src_dir, '/src') into a local before loop. unsure why you do this so add a comment.

Reply:
Done!